src/components/manage/index.js

```diff
@@ -6,11 +6,18 @@ import { CONTRACT_TYPES, TEXT_FIELDS, TOAST, VALIDATION_MESSAGES } from '../../u
 import { InputField } from '../Common/InputField'
 import '../../assets/stylesheets/application.css'
 import { WhitelistInputBlock } from '../Common/WhitelistInputBlock'
-import { successfulFinalizeAlert, successfulDistributeAlert, successfulUpdateCrowdsaleAlert, warningOnFinalizeCrowdsale } from '../../utils/alerts'
+import {
+  successfulFinalizeAlert,
+  successfulDistributeAlert,
+  successfulUpdateCrowdsaleAlert,
+  warningOnFinalizeCrowdsale,
+  notTheOwner
+} from '../../utils/alerts'
 import { getNetworkVersion, sendTXToContract, attachToContract, calculateGasLimit } from '../../utils/blockchainHelpers'
 import { getWhiteListWithCapCrowdsaleAssets, toast } from '../../utils/utils'
 import { contractsInfo, getTiers, processTier, updateTierAttribute } from './utils'
 import { Loader } from '../Common/Loader'
+import web3Store from '../../stores/Web3Store'
```

Comment:
Do we need it here? We injected `web3Store` below.

Reply:
Unrelated to this PR, but we could use something like this https://github.com/coopermaruyama/react-web3 instead of a store, since stores should have plain data, not a complex object like web3. Besides, using a component like that, we'd be sure that web3 is present, available and unlocked when the component is rendered.

Reply:
@vbaranov was imported by the IDE, just deleted it. Thanks for the catch!